opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -249,19 +249,40 @@ namespace Opm {
         return m_minpvValue;
     }
 
+    /// static helper function checking that _one_ coordinate differs with 1
+    bool ijk_are_adj(int i1, int j1, int k1, int i2, int j2, int k2) {
```

Comment:
I meant `free`, but it should probably also be a static function, or what?  I don't intend it to be used from anywhere else than `cartesianAdjacent`.